### PR TITLE
docs: Add repo.lock troubleshooting steps to readme

### PR DESCRIPTION
**Pull request by SgtPooki**
repo.lock issues are frequent, and we don't have an easy place for users to find a solution for this. This PR should help.

### CODE REVIEW

**Comment by SgtPooki** [comment]
@hacdias lidel should be back on Monday, but it would be nice to close some PRs on this repo if you get a chance to peek at this tiny PR.

File: README.md [hunk 1]
```diff
@@ -212,6 +212,14 @@ By default, IPFS Desktop starts the IPFS daemon with the flags `--migrate=true -
 
 You can change this in the IPFS Desktop config file by selecting `Open Configuration File` from the `Advanced` submenu.
 
+### I got a repo.lock error. How do I resolve this?
```
Comment (hacdias) [inline README.md:215]:
```suggestion
### I got a `repo.lock` error. How do I resolve this?
```

File: README.md [hunk 2]
```diff
@@ -212,6 +212,14 @@ By default, IPFS Desktop starts the IPFS daemon with the flags `--migrate=true -
...
+
+In general, this means that a previous process was unable to remove the repo lock (indicator that file is in use) from the repo file. This is supposed to be handled automatically, but sometimes it isn't. If you get this error, you can generally safely delete this file after shutting down any running ipfs daemon's or applications. Simple process is as follows:
```
Comment (hacdias) [inline README.md:217]:
```suggestion
In general, this means that a previous process was unable to remove the repository lock (indicator that file is in use) from the repository directory. This is supposed to be handled automatically, but sometimes it isn't. If you get this error, you can generally safely delete this file after shutting down any running IPFS daemon's or applications. Simple process is as follows:
```

File: README.md [hunk 3]
```diff
@@ -212,6 +212,14 @@ By default, IPFS Desktop starts the IPFS daemon with the flags `--migrate=true -
...
+
+1. Stop ipfs processes
+2. Manually delete lock file
```
Comment (hacdias) [inline README.md:220]:
```suggestion
2. Manually delete lock file, located within the [repository](#how-does-ipfs-desktop-select-the-ipfs-repo-location)
```

File: README.md [hunk 4]
```diff
@@ -212,6 +212,14 @@ By default, IPFS Desktop starts the IPFS daemon with the flags `--migrate=true -
...
+3. Attempt to start ipfs desktop (or other process that received the repo.lock error) again.
```
Comment (hacdias) [inline README.md:221]:
```suggestion
3. Attempt to start ipfs desktop (or other process that received the `repo.lock` error) again.
```

**Review by hacdias — CHANGES_REQUESTED** [review]
Some small nitpicks and a link to the repository location information. I think we can merge this without @lidel's review as it is very small and just doc-related.

File: README.md (hunk 3 above)
Comment (SgtPooki) [reply README.md:220]: ooh nice.

**Comment by SgtPooki** [comment]
Thanks @hacdias ! I used all your suggestions =)

**Review by hacdias — APPROVED** [review]

**Pull request merged**